cmake/FindLz4.cmake

```diff
@@ -71,9 +71,25 @@ find_path(Lz4_SRC_DIR_LZ4
 if (Lz4_SRC_DIR_LZ4)
   get_filename_component(Lz4_SRC_DIR_PARENT ${Lz4_SRC_DIR_LZ4} DIRECTORY)
 
+  set(LZ4_CMAKE_PATH "")
```

Comment:
As a wild idea - what if check here if `LZ4_CMAKE_PATH` is empty sting? If it is not (e.g. someone set it via Cmake parameters) just use provided value. And only if it is not set - do the search?

Reply:
AFAIK we don't set LZ4_CMAKE_PATH previously. I am not sure it is really useful